src/03_modules.py

```diff
@@ -10,22 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv[0])
```

Comment:
This wants one arg per line. argv will be a list, as you recognized here by calling argv[0]. You can iterate through a list to print each element out one at a time.